tests/foreman/ui/test_discoveredhosts.py

```diff
@@ -14,18 +14,18 @@ class Discovery(UITestCase):
     """Implements Foreman discovery tests in UI."""
 
     name = gen_string("alpha")
-    image_path = '/var/lib/libvirt/images/{0}.img'.format(name)
 
     def _pxe_boot_host(self, mac):
         """PXE boot a unknown host"""
-        libvirt_server = 'qemu+tcp://{0}:16509/system'.format(
-            settings.server.hostname)
-        ssh.command('virt-install --hvm --network=bridge:virbr1, --mac={0} '
-                    '--pxe --name {1} --ram=1024 --vcpus=1 --os-type=linux '
-                    '--os-variant=rhel7 --disk path={2},size=10 --connect {3} '
-                    '--noautoconsole'
-                    .format(mac, self.name, self.image_path, libvirt_server))
-        sleep(30)
+        ssh.command(
+            u'virt-install --hvm --network=bridge:{0}, --mac={1} '
+            '--pxe --name {2} --ram=1024 --vcpus=1 --os-type=linux '
+            '--os-variant=rhel7 --disk path={3},size=10 --noautoconsole'
+            .format(settings.vlan_networking.bridge, mac, self.name,
+                    self.image_path),
+            hostname=self.libvirt_host
+        )
+        sleep(70)
```

Comment:
I believe at this stage there is no way for us to know the ip of the "to be discovered host" and hence we can't check whether the "to be discovered host" is up and hence the `sleep` here ? Can you confirm the same please ?